app/views/site/id.html.erb

```diff
@@ -3,9 +3,7 @@
 <head>
   <meta charset='utf-8'>
   <%= stylesheet_link_tag "id" %>
-  <!--[if !IE || gte IE 9]><!-->
   <%= javascript_include_tag "id" %>
-  <!-- <![endif]-->
```

Comment:
Does removing this conditional make sense as part of this PR? Looks like something that should be removed, but perhaps in a separate PR.

Reply:
Ok, I extracted the obsolete IE code and will open a separate PR for it. -> #6489